bin/MadGraph5_aMCatNLO/cards/production/13TeV/VV/WWTo4Q_5f_amcatnloFxFx/CP5_5f_max1j_FxFx_qCut25.py

```diff
@@ -0,0 +1,39 @@
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.Pythia8aMCatNLOSettings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
```

Comment:
should we start to include concurrent as baseline ?

Reply:
ahh OK will add the externalLHEproducer as well then

Reply:
I removed it since I think we can discuss the details in fragments later when we have the gridpacks